src/dcat-ap/dcat-formatters.test.ts

```diff
@@ -203,91 +195,154 @@ describe('formatDcatDataset', () => {
         'dct:title': 'OGC WMS',
       },
     };
-    const result = JSON.parse(formatDcatDataset(distDataset, defaultFormatTemplate));
```

Comment:
This looks like a big diff, but if you open in VSC mode (hit `.`), you'll see that I moved all of this into a nested `distributions` describe.

Reply:
Nice, well thought out